### DISCUSSION TITLE
How to train structures with non-periodic boundary along one direction?

### DISCUSSION THREAD

**Discussion by yunai2384** [deepmodeling/deepmd-kit#2454] [Q&A]
Dear all, i know how to train clusters with non-periodic boundary condition, while is there anyone can tell me how to train structures with  non-periodic conditions applied to only one direction?  Thanks.

**Comment by wanghan-iapcm** [comment]
https://docs.deepmodeling.com/projects/deepmd/en/master/data/data-conv.html#numpy-format

**Reply by yunai2384** [reply]
Thank you for your timely response, i undersand it if you mean that the nopbc file can be applied even the other two directions are periodic,  thank you!

**Comment by njzjz** [comment]
You can give a large box size in that direction.
